packages/mgt-components/src/components/mgt-profile/mgt-profile.ts

```diff
@@ -306,7 +306,9 @@ export class MgtProfile extends BasePersonCardSection {
                  ${position?.detail?.company?.displayName}
                </div>
                <div class="work-position__location" tabindex="0">
-                 ${position?.detail?.company?.address?.city}, ${position?.detail?.company?.address?.state}
+                 ${position?.detail?.company?.address?.city}${
+          position?.detail?.company?.address?.city && position?.detail?.company?.address?.state ? ', ' : ''
+        }${position?.detail?.company?.address?.state}
```

Comment:
Can we refactor this to use a helper function? nested ternaries like this are hard to read, and with lots of optional chaining like this there's a lot of repeated checks on the same property.

```TypeScript
const displayLocation = (position: PhysicalAddress | undefined): string => {
  if (address?.city) {
    if (address.state) {
     return `${address.city}, ${address.state}`;
    }
    return address.city;
  } 
  return nothing;
}
```